common/replication/block_fetcher_test.go

```diff
@@ -801,6 +804,84 @@ func TestBlockFetcherMaxRetriesExhausted(t *testing.T) {
 	require.Equal(t, (*common.Block)(nil), bf.PullBlock(1))
 }
 
+func TestBlockFetcherInitVerifierFromGenesisBlock(t *testing.T) {
+	// In BlockFetcher, if VerifyBlock is set to nil, BlockFetcher should connect to orderers and pull
+	// genesis block and set the compute the VerifyBlock function from it.
+	// So BlockFetcher can configure itself implicitly by pulling genesis block in a bft setting
+
+	t.Parallel()
+	bf := replication.BlockFetcher{}
+
+	// Create a genesis block
+	confSys := genesisconfig.Load(genesisconfig.SampleInsecureSoloProfile, configtest.GetDevConfigDir())
```

Comment:
How does this activate BFT if we use a Solo profile?

Reply:
This just creates a proper genesis block to be pulled by blockfetcher. bft activation depends on MaxByzantineNodes